include/deal.II/base/config.h.in

```diff
@@ -57,6 +57,7 @@
 #cmakedefine DEAL_II_WITH_TRILINOS
 #cmakedefine DEAL_II_WITH_UMFPACK
 #cmakedefine DEAL_II_WITH_ZLIB
+#cmakedefine DEAL_II_TRILINOS_WITH_ZOLTAN
```

Comment:
nitpick, but could you please move this after the line `#cmakedefine DEAL_II_TRILINOS_WITH_ROL` where other `Trilinos`-related things are defined.

Reply:
no problem. Thanks!